addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql

```diff
@@ -0,0 +1,46 @@
+\timing
+
+-- Find packages in multiple locations
+SELECT
+    p.package_id,
+    COUNT(location_id)
+FROM (
+    SELECT
+        package_id,
+        location_id
+    FROM
+        stock_quant
+    WHERE
+        package_id IS NOT NULL
+    GROUP BY
+        package_id,
+        location_id
+) AS p
+GROUP BY
+    p.package_id
+HAVING
+    COUNT(location_id) > 1
+;
+
+-- Find pallets of packages in mutliple locations
+SELECT
+    p.pallet_id,
+    COUNT(location_id)
+FROM (
+    SELECT
+        pallet.id AS pallet_id,
+        q.location_id
+    FROM stock_quant q
+    LEFT JOIN
+        stock_quant_package pack ON pack.id = q.package_id
+    LEFT JOIN
+        stock_quant_package pallet ON pallet.id = pack.package_id
+    GROUP BY
+        pallet.id,
+        q.location_id
+) AS p
+GROUP BY
+    p.pallet_id
+HAVING
+    p.pallet_id IS NOT NULL AND COUNT(location_id) > 1
```

Comment:
I guess miquel wanted this in the where as well?

Reply:
actually can pallet.id be null?

Reply:
I've moved the where inside the select

Reply:
Changed my mind and used the inner join. Good call